### ISSUE TITLE
Remove deprecated `Discussion` component from site and Prismic

### ISSUE THREAD

**Issue by jennpb** [wellcomecollection/wellcomecollection.org#10184]
🛑 Dependent on #10182 and #10183 working effectively. Check whether this deprecation is required/advisable. 🛑 

# What?

After the new `explanatoryText` component is implemented for embedded audio, we no longer need the existing `Discussion` component.

This component is currently used on legacy podcast pages that feature a Soundcloud embed.

We can remove the Discussion component from these pages and make the transcript section open by default.

Examples:
Bidisha podcast https://wellcomecollection.org/articles/YRU4ExEAAFTsyFIH
Mark Thomas podcast https://wellcomecollection.org/articles/XyL-NxAAACAADrLk

# Why? 

No longer needed because we no longer use Soundcloud to host podcasts

# Background  

This came about via https://github.com/wellcomecollection/wellcomecollection.org/issues/10143

See Dom's comment: https://github.com/wellcomecollection/wellcomecollection.org/issues/10143#issuecomment-1698989866

# Done when
-  [x] Legacy transcripts moved to new collapsible content fields
-  [x] Discussion slice removed from Articles content model
-  [x] Discussion component removed from Cardigan
-  [x] Discussion removed from Gitbook Slice page

**Comment by rcantin-w** [comment]
I can bring it up at the next Editorial standup which is tomorrow, where I'll suggest this becomes a Text slice instead. As discussed just now, we won't do exactly what Dom suggested (keeping the grey background) as that would mean it's a different style and therefore still a different component.

**Comment by rcantin-w** [comment]
All done!
